data-access/nexustiles/nexustiles.py

```diff
@@ -55,7 +61,15 @@ def __init__(self, skipCassandra=False, skipSolr=False, config=None):
             self._config = config
 
         if not skipCassandra:
-            self._cass = CassandraProxy(self._config)
+            datastore = self._config.get("datastore", "store")
+            if datastore == "cassandra":
+                self._cass = CassandraProxy(self._config)
+            elif datastore == "s3":
+                self._s3 = S3Proxy(self._config)
+            elif datastore == "dynamo":
+                self._dynamo = DynamoProxy(self._config)
+            else:
+                print("Error reading datastore from config file")
```

Comment:
This should really raise an exception because if we get to this point and there is no usable datastore, we will not be able to use the NexusTileService.